Modules/Crm/Http/Controllers/TasksController.php

```
@@ -2,104 +2,55 @@
+            'filter_method' => 'filter_tasks',
+            'tasks' => $tasks,
+            'task_statuses' => Task::STATUSES,
+        ]);
     }
 
-    /**
-     * Store a newly created resource.
-     */
-    public function store()
+    /** @legacy-file application/modules/tasks/controllers/Tasks.php:50 */
+    public function form(?int $id = null)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)